### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by prateekcr**
Updating README to reflect the changes in this release.

Signed-off-by: Chandrakar, Prateek <prateek.chandrakar@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,52 +82,20 @@ The following instructions follow these notations:
...
-$ ./run-ops
-```
-
-Open a new terminal window and start the device service:
-```
-$ cd <pri-device-dir>
-$ ./device
-```
+For successful onboarding of Ondie ECDSA clients, appropriate CRLs must be present in the path as specified by the property `org.sdo.ops.ondie-ecdsa-material-path`. 
+If the CRLs needs to be downloaded into the same path, set the property `org.sdo.ops.ondie-ecdsa-material-urls` with a list of appropriate URLs from where the CRL will be downloaded.
```
Comment (trbehera) [inline README.md:97]: CRLs => CRL files

needs => need

the CRL will be downloaded => the CRL files will be downloaded

File: README.md [hunk 2]
```diff
@@ -82,52 +82,20 @@ The following instructions follow these notations:
...
+If demo is being run in a closed network or if no update needs to be made to the existing CRLs in the same path, set the property `org.sdo.ops.ondie-ecdsa-material-update` value to 'false'.
```
Comment (trbehera) [inline README.md:98]: CRLs => CRL files

File: README.md [hunk 3]
```diff
@@ -145,7 +113,7 @@ $ ./rendezvous
 Open a new terminal window and start the SDO IoT Platform SDK services:
 ```
 $ cd <sdo-iot-platform-sdk-root>/demo
-$ sudo docker-compose up
+$ docker-compose up
```
Comment (trbehera) [inline README.md:116]: docker-compose up --build ?

File: README.md [hunk 4]
```diff
@@ -157,4 +125,58 @@ $ ./device
 NOTE: To re-trigger TO0 for a device for which TO0 was previously done and is currently active,
 please delete the file 'state.json' located at <ocs-config-dir>/db/v1/devices/<deviceID>/.
 
+## Configuring properties in SDO Iot Platform SDK
```
Comment (trbehera) [inline README.md:128]: Iot => IoT ?

File: README.md (hunk 1 above)
Comment (prateekcr) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (prateekcr) [reply README.md]: Updated.

File: README.md (hunk 4 above)
Comment (prateekcr) [reply README.md]: Updated.

File: README.md (hunk 3 above)
Comment (prateekcr) [reply README.md]: Haven't been using the build flag in the previous releases. Added it now.

**Pull request merged**